scripts/animal.coffee

```diff
@@ -9,12 +9,14 @@
 #
 
 cow = require('cowsay')
-cows = ['cheese','bunny','dragon','elephant','mule','octopus',
-        'flaming-sheep','goat','ghostbusters','hedgehog','kiwi',
-        'kitty', 'mech-and-cow','moose','khilesh','lobster',
-        'sheep','small','squirrel','stegosaurus','bees','bijan',
-        'turtle','whale','elephant-in-snake','moofasa','owl',
-        'skeleton','turkey','tux','mail','cat', 'armadillo','shrek']
+#cows = ['cheese','bunny','dragon','elephant','mule','octopus',
+#        'flaming-sheep','goat','ghostbusters','hedgehog','kiwi',
+#        'kitty', 'mech-and-cow','moose','khilesh','lobster',
+#        'sheep','small','squirrel','stegosaurus','bees','bijan',
+#        'turtle','whale','elephant-in-snake','moofasa','owl',
+#        'skeleton','turkey','tux','mail','cat', 'armadillo','shrek']
+
+cows = ['skeleton','ghostbusters']
```

Comment:
Maybe it'd be better to check the date here and use the spooky list if Halloween?